src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -143,58 +140,69 @@ core::smart_refctd_ptr<ICPUImage> createAndconvertImageData(ICPUImage::SCreation
 
```

Comment:
if you're going to use `asset::IImageAssetHandlerBase::calcPitchInBlock` as a value any time that a "TEXEL COUNT VALUE IS REQUIRED" the least you could do is assert that the block size is 1x1x1

Reply:
but it is in a loader, nobody will use it and we are sure it isn't BC

Reply:
okay